curl_cffi/requests/session.py

```diff
@@ -279,10 +281,7 @@ def _set_curl_options(
         c = curl
 
         # method
-        if method == "POST":
-            c.setopt(CurlOpt.POST, 1)
-        elif method != "GET":
-            c.setopt(CurlOpt.CUSTOMREQUEST, method.encode())
+        c.setopt(CurlOpt.CUSTOMREQUEST, method.encode())
```

Comment:
`CURLOPT_POST` is not the same as `CURLOPT_CUSTOMREQUEST` with "POST", see #135

Reply:
Fixed https://github.com/yifeikong/curl_cffi/pull/284/commits/1aa8c23c67408012b0f3c538e80ac0408adb71e3